ext/mixed/js/api.js

```diff
@@ -15,307 +15,212 @@
+        kanjiFind(text, optionsContext) { return this._invoke('kanjiFind', {text, optionsContext}); }
+        definitionAdd(definition, mode, context, details, optionsContext) { return this._invoke('definitionAdd', {definition, mode, context, details, optionsContext}); }
+        definitionsAddable(definitions, modes, context, optionsContext) { return this._invoke('definitionsAddable', {definitions, modes, context, optionsContext}); }
+        noteView(noteId) { return this._invoke('noteView', {noteId}); }
+        templateRender(template, data) { return this._invoke('templateRender', {data, template}); }
+        audioGetUri(definition, source, details) { return this._invoke('audioGetUri', {definition, source, details}); }
+        commandExec(command, params) { return this._invoke('commandExec', {command, params}); }
+        screenshotGet(options) { return this._invoke('screenshotGet', {options}); }
+        sendMessageToFrame(frameId, action, params) { return this._invoke('sendMessageToFrame', {frameId, action, params}); }
+        broadcastTab(action, params) { return this._invoke('broadcastTab', {action, params}); }
+        frameInformationGet() { return this._invoke('frameInformationGet'); }
+        injectStylesheet(type, value) { return this._invoke('injectStylesheet', {type, value}); }
+        getEnvironmentInfo() { return this._invoke('getEnvironmentInfo'); }
+        clipboardGet() { return this._invoke('clipboardGet'); }
+        getDisplayTemplatesHtml() { return this._invoke('getDisplayTemplatesHtml'); }
+        getQueryParserTemplatesHtml() { return this._invoke('getQueryParserTemplatesHtml'); }
+        getZoom() { return this._invoke('getZoom'); }
+        getDefaultAnkiFieldTemplates() { return this._invoke('getDefaultAnkiFieldTemplates'); }
+        getAnkiDeckNames() { return this._invoke('getAnkiDeckNames'); }
+        getAnkiModelNames() { return this._invoke('getAnkiModelNames'); }
+        getAnkiModelFieldNames(modelName) { return this._invoke('getAnkiModelFieldNames', {modelName}); }
+        getDictionaryInfo() { return this._invoke('getDictionaryInfo'); }
+        getDictionaryCounts(dictionaryNames, getTotal) { return this._invoke('getDictionaryCounts', {dictionaryNames, getTotal}); }
+        purgeDatabase() { return this._invoke('purgeDatabase'); }
+        getMedia(targets) { return this._invoke('getMedia', {targets}); }
+        log(error, level, context) { return this._invoke('log', {error, level, context}); }
+        logIndicatorClear() { return this._invoke('logIndicatorClear'); }
+        importDictionaryArchive(archiveContent, details, onProgress) { return this._invokeWithProgress('importDictionaryArchive', {archiveContent, details}, onProgress); }
+        deleteDictionary(dictionaryName, onProgress) { return this._invokeWithProgress('deleteDictionary', {dictionaryName}, onProgress); }
+        modifySettings(targets, source) { return this._invoke('modifySettings', {targets, source}); }
```

Comment:
I did it this way because:
* Using newlines (same layout as before) basically requires 4x the additional lines.
* Spaces between `)` and `{` would require changes to every line if a new function is added with long arguments, or if a function's arguments are changed.

I'll probably just revert back to the newline layout.

Reply:
930cdac51fc30586981ecbd7b038473638d13928
f864a58f5e7fd82b2c70b8ca2846d9a567eea309